### ISSUE TITLE
TO_DATE() with colons causes binds miscount.

### ISSUE THREAD

**Issue by NormanDunbar** [NormanDunbar/TraceMiner2#9]
A statement like:

````
INSERT  INTO table_name
(operation$,cscn$,commit_timestamp$,rsid$,row_id$,username$,timestamp$,"SALE_PSN","CUST_ORD_ID","CUST_ID","ORD_DTE","TOT_AMT_PNC","ORD_STAT","LD_TMESTP")
VALUES  (:op,:cscn,   to_date('4000-01-01:00:00:00','YYYY-MM-DD:HH24:MI:SS'),:seq,:m,:un,SYSDATE,:1,:2,:3,:4,:5,:6,:7);
````
Parses out too many bind variables because of the value & format strings in the TO_DATE() call.

Consider also:

- TO_CHAR
- TO_DATE
- TO_DSINTERVAL
- TO_NCHAR
- TO_NUMBER
- TO_TIMESTAMP
- TO_TIMESTAMP_TZ
 as these all have a format string that could take colons. Sigh.

**Comment by NormanDunbar** [comment]
This trace file demonstrates the problem.
[dates.zip](https://github.com/NormanDunbar/TraceMiner2/files/2128099/dates.zip)

**Comment by NormanDunbar** [comment]
Resolved by Commit 2161c51 . (Regex systems) and commit 1b484ef (non-regex systems).
Basically, when we look for a bind, it needs to have a character preceding the colon, which is:

space, tab, newline, ( = - + / * ,

Anything else, is not a bind.

(Seems to work!)
